src/ui/server/controllers/insurance/policy/multiple-contract-policy/validation/rules/total-months-of-cover.ts

```diff
@@ -23,13 +26,36 @@ const {
   },
 } = ERROR_MESSAGES;
 
+const MINIMUM = 1;
+export const MAXIMUM = APPLICATION.POLICY.TOTAL_MONTHS_OF_COVER;
+
 /**
  * totalMonthsOfCoverRules
  * Returns the result of emptyFieldValidation
  * @param {Express.Response.body} Express response body
  * @param {Object} Errors object from previous validation errors
  * @returns {Object} Validation errors
  */
-const totalMonthsOfCoverRules = (formBody: RequestBody, errors: object) => emptyFieldValidation(formBody, FIELD_ID, ERROR_MESSAGE.IS_EMPTY, errors);
+const totalMonthsOfCoverRules = (formBody: RequestBody, errors: object) => {
+  if (!objectHasProperty(formBody, FIELD_ID)) {
+    return emptyFieldValidation(formBody, FIELD_ID, ERROR_MESSAGE.IS_EMPTY, errors);
+  }
+
+  if (!isNumber(formBody[FIELD_ID])) {
+    return generateValidationErrors(FIELD_ID, ERROR_MESSAGE.INCORRECT_FORMAT, errors);
+  }
+
+  const numb = Number(formBody[FIELD_ID]);
```

Comment:
different name? numb just doesn't look right to me

Reply:
Renamed to `monthsOfCover`